documentation/identifiers.md

```diff
@@ -76,67 +86,80 @@ There are many different kinds of organisations that give, receive or benefit fr
+* Local authorities - [GB-LAE](http://org-id.guide/list/GB-LAE) (England), [GB-LAS](http://org-id.guide/list/GB-LAS) (Scotland), [GB-PLA](http://org-id.guide/list/GB-PLA) (Wales)  - 
+
+If you have a registered number from some other scheme, including overseas registrars, check the [org-id List Locator](http://org-id.guide/) for a Registration Agency Code to use. If the Registration Agency Code you need is not listed, [contact the support team](http://www.threesixtygiving.org/contact/).
+
+If you do not have any external registration numbers for the organisation, use your 360Giving prefix and any internal identifier you have for this organisation.
 
-    If you have a registered number from some other scheme, including overseas registrars, check the [IATI Organisation Registration Agency Codelist](http://iatistandard.org/codelists/OrganisationRegistrationAgency/) for a Registration Agency Code to use. If the Registration Agency Code you need is not listed, [contact the support team](http://www.threesixtygiving.org/contact/).
 
-5. **No registered number**
+```eval_rst
+
+.. admonition:: Special 360 giving fields for charity and company number
 
-    If you do not have any external registration numbers for the organisation, use your 360Giving prefix and any internal identifier* you have for this organisation.
-    
-    *If you use a database that records details of organisations in a separate lookup table, this may provide an identifier you can use. 
-    
-    If you only record data in a spreadsheet, and don't assign organisations an ID, you could use a spreadsheet formula to turn the organisation name into an identifier (e.g. removing spaces and lowercasing the name). The support team can provide guidance on this. 
-    
-    If there is a chance that your organisation identifiers might overlap with grant identifiers, just add 'ORG' into the identifier string (e.g. '360G-xyztrust-ORG123')
-    
-### Company Numbers and Charity Numbers
+   Because 'Company Number' and 'Charity Number' are so important for analysing grantmaking in the UK, the 360 Giving Standard includes additional special fields these on their own (without the prefixes), to help users of the data. 
+
+   If you have these details, you should fill them in, **in addition to** providing the unique organisation identifier using the method above. 
+```
 
-Sometimes you may have recorded both the company number, and charity number, of an organisation in your data. Because having both of these is important, 360Giving also includes two extra separate fields for ```Company Number``` and ```Charity Number```. If you have these details, you should fill them in, in addition to providing the unique organisation identifier using the method above. 
+### Collecting organisation identifiers
 
+Make sure that when you collect charity or company numbers, you clearly identify in your forms or systems which type of number they are. 
+
+```eval_rst
+.. _codes:
+```
 
-## Codes
+## Classifications
```

Comment:
This feels out of scope - we should avoid giving pointers about how they organise their grantmaking systems - in this guidance.

Reply:
I mean the bit about how numbers are collected, and classifications, btw!

Reply:
I don't know, because I didnt see so much use of classifications.

Reply:
Not v. strong view on classifications. Commenting because our support on this is patchy, so it is odd to see it crop up here.

I see this guidance as explaining how to format the data they have to 360, so ending it with a 'make sure you do' comment about how they manage their internal systems feels like a leap. I know there are a grantmaking systems where there isn't a choice to configure this way. If we keep it, need to soften to 'if you can, it's helpful to etc'.